onnxruntime/core/providers/nnapi/nnapi_builtin/nnapi_execution_provider.cc

```diff
@@ -243,6 +251,9 @@ common::Status NnapiExecutionProvider::Compile(const std::vector<FusedNodeAndGra
   for (const auto& fused_node_and_graph : fused_nodes_and_graphs) {
     Node& fused_node = fused_node_and_graph.fused_node;
     const onnxruntime::GraphViewer& graph_viewer(fused_node_and_graph.filtered_graph);
+    if (graph_viewer.GetOutputs().empty()) {
+      continue;
+    }
```

Comment:
Is there an actual use case where this happens? I wouldn't add it unless you can explicitly test it. The `continue` will result in a gap in node_compute_funcs which could cause a runtime error.

Reply:
Not yet. But I am supposing there would be a case happend.

Have moved to the `GetCapability()`